### ISSUE TITLE
Drop JSON configs in favor of YAML

### ISSUE THREAD

**Issue by JackuB** [Snyk/broker#7]
My reason would be that JSON is more machine readable and we want these configs to be easier to audit. Compare this part of config:

```json
{
  "public":
  [
      {
        "//": "used for pushing up webhooks from github",
        "method": "POST",
        "path": "/webhook/github",
        "valid": [
          {
            "//": "accept all pull request state changes (these don't have files in them)",
            "path": "pull_request.state",
            "value": "open"
          },
          {
            "path": "commits.*.added.*",
            "value": "package.json"
          },
          {
            "path": "commits.*.modified.*",
            "value": "package.json"
          },
          {
            "path": "commits.*.added.*",
            "value": ".snyk"
          },
          {
            "path": "commits.*.modified.*",
            "value": ".snyk"
          }
        ]
      }
  ]
}
```

```yaml
public:
# used for pushing up webhooks from github
- method: POST
  path: "/webhook/github"
  valid:
  # accept all pull request state changes (these don't have files in them)
  - path: pull_request.state
    value: open
  - path: commits.*.added.*
    value: package.json
  - path: commits.*.modified.*
    value: package.json
  - path: commits.*.added.*
    value: ".snyk"
  - path: commits.*.modified.*
    value: ".snyk"
```

Thoughts?

**Comment by guypod** [comment]
No objection here, though I don't feel it's much of a priority. 
If you wanted to do a PR, we wouldn't object.

**Comment by JackuB** [comment]
Resolved by #10
